### PR TITLE
Add Unit Tests for Core.Objects

### PR DESCRIPTION
**Pull request by pmosk**

### CODE REVIEW

File: src/PrimeFuncPack.Core/core_objects/Objects.Tests/StringPredicateExtensionsTests/StringPredicateExtensionsTests.OrWhiteSpace.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            string source = string.Empty;
+
+            var actual = source.IsNullOrWhiteSpace();
+            Assert.True(actual);
+        }
+
+        [Test]
+        [TestCase(" ")]
+        [TestCase("   ")]
+        public void IsNullOrWhiteSpace_SourceIsWhiteSpace_ExpectTrue(
```
Comment (andreigit) [inline src/PrimeFuncPack.Core/core_objects/Objects.Tests/StringPredicateExtensionsTests/StringPredicateExtensionsTests.OrWhiteSpace.cs:31]: Shall we add some whitespace test cases different from just SPACE - like TAB ("\t")?

File: src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>PrimeFuncPack.Core.Objects.Tests</RootNamespace>
+    <AssemblyName>PrimeFuncPack.Core.Objects.Tests</AssemblyName>
```
Comment (andreigit) [inline src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj:8]: Shall we add Authors/Copyright tags? Tests are not supposed to be published as packages, but just to protect the sources

File: src/PrimeFuncPack.Core/core_objects/Objects.Tests/StringPredicateExtensionsTests/StringPredicateExtensionsTests.OrWhiteSpace.cs (hunk 1 above)
Comment (pmosk) [reply src/PrimeFuncPack.Core/core_objects/Objects.Tests/StringPredicateExtensionsTests/StringPredicateExtensionsTests.OrWhiteSpace.cs:32]: Have done

File: src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj (hunk 2 above)
Comment (pmosk) [reply src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj:11]: Done

File: src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <IsPackable>false</IsPackable>
+    <Description>PrimeFuncPack: A Functional Programming Pack for .NET</Description>
+    <Copyright>Copyright © 2020 Andrei Sergeev, Pavel Moskovoy</Copyright>
+    <RootNamespace>PrimeFuncPack.DependencyPipeline</RootNamespace>
```
Comment (andreigit) [inline src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj:9]: Seems two different RootNamespace tags in the same csproj

Comment (pmosk) [reply src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj]: Fixed

**Pull request merged**